include/dt-bindings/dma/stm32_dma.h

```diff
@@ -4,8 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 /* macros for channel-config */
+#define STM32_DMA_CONFIG_DIRECTION(config)		(config & 0x3 << 6)
 #define STM32_DMA_CONFIG_PERIPH_ADDR_INC(config)	(config & 0x1 << 9)
 #define STM32_DMA_CONFIG_MEM_ADDR_INC(config)		(config & 0x1 << 10)
+#define STM32_DMA_CONFIG_PERIPH_DATA_SIZE(config)	(config & (0x3 << 11))
+#define STM32_DMA_CONFIG_MEM_DATA_SIZE(config)		(config & (0x3 << 13))
```

Comment:
This should come with the equivalent yaml binding.

Reply:
moved, see commit 383b94d117c7cbe43a5892b2ff13b99955b97f52